x-pack/plugins/security_solution/public/overview/components/overview_host_stats/index.tsx

```diff
@@ -245,8 +245,20 @@ const hostStatGroups: StatGroup[] = [
   },
 ];
 
-const Title = styled.div`
+const MoveItLeftTitle = styled.div`
   margin-left: 24px;
+  @media only screen and (min-width: ${({ theme }) => theme.eui.euiBreakpoints.m}) {
+    max-width: 40px;
+  }
+`;
+const MoveItLeft = styled.div`
+  margin-left: 24px;
+`;
+
+const NoMarginTopFlexItem = styled(EuiFlexItem)`
+  @media only screen and (max-width: ${({ theme }) => theme.eui.euiBreakpoints.m}) {
+    margin-top: -10px !important;
```

Comment:
just wondering if there's a conflict somewhere that `!important` overrides, or is this just ot make sure it doesn't get overridden in the future?

Reply:
yes there is an important we're overwriting